src/Framework/Menus/CustomBoard.cs

```diff
@@ -16,9 +16,10 @@ class CustomBoard : IClickableMenu
     {
         public static readonly Dictionary<string, Quest> todayQuests;
 
-        private readonly ClickableComponent _acceptQuestButton;
+        public ClickableComponent AcceptQuestButton;
```

Comment:
make it `readonly` (field value is assigned only in constructor) and because it's field and not property, use `camelCase` instead of `PascalCase`, so it will be called as `acceptQuestButton`

```suggestion
        public readonly ClickableComponent acceptQuestButton;
```

Reply:
will do